cypress/support/utils.js

```diff
@@ -153,6 +153,9 @@ export const assertCodePaneLineCount = (lines) => {
   getCodeEditor().within(() =>
     cy.get('.CodeMirror-line').should('have.length', lines)
   );
+
+  // Wait after check to ensure original focus is restored
+  cy.wait(500); // eslint-disable-line @finsit/cypress/no-unnecessary-waiting
```

Comment:
Is this needed for every call of `assertCodePaneLineCount`, or does it only affect a specific one?

Reply:
It's needed for at least a couple and it's used 10 times total. The tests aren't always consistent but I'll make it an optional prop anyway to be more explicit when the wait is needed.